ray-operator/test/support/environment.go

```diff
@@ -29,3 +31,25 @@ func lookupEnvOrDefault(key, value string) string {
 	}
 	return value
 }
+
+func GetRayVersionForAutoScalerV2() string {
+	rayVersion := os.Getenv("E2E_AUTOSCALER_V2_TEST_RAY_VERSION")
+	if strings.TrimSpace(rayVersion) == "" {
+		rayVersion = "2.10.0"
+	}
+	return rayVersion
+}
+
+func GetRayImageForAutoScalerV2() string {
+	rayImage := os.Getenv("E2E_AUTOSCALER_V2_TEST_RAY_IMAGE")
+	if strings.TrimSpace(rayImage) == "" {
+		rayImage = "rayproject/ray:2.10.0.ee9422-py310"
+	}
+	// detect if we are running on arm64 machine, most likely apple silicon
+	// the os name is not checked as it also possible that it might be linux
+	// also check if the image does not have the `-aarch64` suffix
+	if runtime.GOARCH == "arm64" && !strings.HasSuffix(rayImage, "-aarch64") {
+		rayImage = rayImage + "-aarch64"
+	}
+	return rayImage
+}
```

Comment:
I noticed that after changing to 2.39.0, the CI failed. This might be challenging for a first-time contributor to handle, so I created a new issue to resolve it myself. Once I’ve fixed this issue, you can rebase the master branch. Until then, let's put this PR on hold.

https://github.com/ray-project/kuberay/issues/2595

Reply:
Hey @MortalHappiness, thank you for creating another issue for the version update.
I can share what I found in my attempts:

1. The TestRayServiceInPlaceUpdate failure may not be directly related to the image/project version. Mine failed on both 2.9.0 (once) and 2.39.0 (multiple attempts).
2. There are some hard-coded "2.9.0" and "2.10.0" in the code, which may require refactoring.

Reply:
Yes I found that it is because of `TestRayServiceInPlaceUpdate` is flaky, not related to the version. I'll change the issue title.